modules/installation-special-config-kmod.adoc

```diff
@@ -169,14 +169,7 @@ $ sudo kmods-via-containers build simple-kmod $(uname -r)
 +
 [source,terminal]
 ----
-$ sudo systemctl enable kmods-via-containers@simple-kmod.service
-----
-
-.. Start the service:
-+
-[source,terminal]
-----
-$ sudo systemctl start kmods-via-containers@simple-kmod.service
+$ sudo systemctl enable kmods-via-containers@simple-kmod.service --now
```

Comment:
Also remove line 168 - since there's only one command now, line 167 is the best description for this action.